automation/CommandSequence.py

```diff
@@ -42,6 +43,12 @@ def __init__(self, url: str, reset: bool = False,
         site_rank : int, optional
             Integer indicating the ranking of the page to visit, saved
             to `site_visits`
+        callable :
+            A callback to be invoked once all data regarding this
+            CommandSequence has been saved out or it has been interrupted
+            The function will be passed a bool that's true if everything has
+            been successfully saved out if it's false something bad happened
+            and no or only incomplete data may have been saved out
```

Comment:
```suggestion
            If the command sequence completes and all data is saved successfully, `True` will be passed to the callback.
            Otherwise `False` will be passed. A value of `False` indicates that the data saved from the site visit may be 
            incomplete or empty.
```